data/gettingstarted.html

```diff
@@ -1281,7 +1325,11 @@ <h3>&zwnj;Window shortcuts - general&zwnj;</h3>
                   <i class="material-icons">&#xE145;</i>
                   <b class="btn btn-xs disabled">S</b>
                 </td>
-                <td>&zwnj;Call Window Shuffler gui (window tiling grid).&zwnj;</td>
+                <td>&zwnj;Call Window Shuffler gui (window tiling grid).&zwnj;
```

Comment:
Formatting has gone a bit awry.  If you are using vscode to edit - then I would recommend the beautify extension to reformat the code.

Reply:
Done, thank you.